ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -157,11 +157,15 @@ const sectionTitle = ({ fieldTitle, section }) => (
 const HOM_SECTION = 'Homoplasmy'
 const HET_SECTION = 'Heteroplasmy'
 
+const SV_CALLSET_POP = { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE }
 const CALLSET_POP = { field: 'callset', fieldTitle: 'This Callset', acDisplay: 'AC' }
+const SEQR_POP = { ...CALLSET_POP, field: 'seqr', fieldTitle: 'seqr' }
 
 const POPULATIONS = [
-  { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE },
+  SV_CALLSET_POP,
+  { ...SV_CALLSET_POP, field: 'sv_seqr', fieldTitle: 'seqr' },
```

Comment:
Same as above for the `fieldTitle` field.

Reply:
Its very clear in the UI that this is referring to a population frequency. Adding the word "Callset" is really inaccurate or misleading - it implies all the samples in seqr have been called together, which they haven't. And adding the words "population" or "frequency" clutters to UI without adding any information to the user

Reply:
I see. But the statement `all the samples in seqr have been called together` confuses me. What does the word `called` mean exactly?

Reply:
It means that they were run through a joint calling pipeline, like GATK, which is how a joint called VCF is generated